server/protocols/webadmin/webadmin-data/src/main/java/org/apache/james/webadmin/dto/MappingValueDTO.java

```diff
@@ -26,15 +26,15 @@
     private final String type;
     private final String mapping;
 
-    public MappingValueDTO(Mapping mapping) {
-        this(mapping.getType().name(), mapping.getMappingValue());
-    }
-
     public MappingValueDTO(String type, String mapping) {
         this.type = type;
         this.mapping = mapping;
     }
 
+    public static MappingValueDTO fromMapping(Mapping mapping) {
```

Comment:
static methods should always come before non-static ones.
You should try to run checkstyle with our configuration (located at project root) to avoid such errors.